style.css

```diff
@@ -70,10 +109,11 @@ body {
 .List-section-hr {
   border: 1px solid #789;
   width: 40%;
-  margin-top: 20px;
+  margin: 20px 0;
 }
```

Comment:
- [ ] As the provided wireframe is, we should try to make our version look alike as much as we can, Your list section is not aligned in the middle and hence that gives a bad user experience, 

|orginal wirframe|Your design version|
|---|---|
|![Screenshot from 2021-09-16 17-46-58](https://user-images.githubusercontent.com/55918152/133645591-7f83b48c-53cb-4728-8ed0-38ed857f2aff.png)|![Screenshot from 2021-09-16 17-46-17](https://user-images.githubusercontent.com/55918152/133645539-8f39542b-3b9e-458c-9665-e9f491c0975e.png)|

_Note: Please notice that the middle part of your list is not well aligned vertically as it designed in the original wireframe_ => Would you address that?